samples/boards/reel_board/mesh_badge/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _mesh_badge:
+
+Mesh Badge
+##########
+
+Overview
+********
+
+A sample app for the reel board that showcases Bluetooth Mesh
+
+The app starts off as a regular Bluetooth GATT peripheral application. It's
+recommended to install the "nRF Connect" app on your phone (available both for
+Android and iOS) to access the service that the app exposes.
+
+Steps to set up
+***************
+
+#. Scan for devices and look for "reel board"
+#. Connect to the device. You'll see a single service - select it
+#. Request to write to the characteristic by pressing on the upward pointing
+   arrow symbol
+#. Select "Text" to enter text instead of hex
+#. Enter your name (or any other random data). Multiple words separated by
+   spaces are possible. The font used enables three rows of up to 12 characters
+   wide text. You can force line breaks with a comma.
+#. Press "Send" - this will trigger pairing since this is a protected
+   characteristic. The passkey for the pairing will be shown on the board's
+   display.
```

Comment:
I presume you need to do something in the app to complete the paring using the passkey?

Reply:
yes, you need to enter it in the phone - I added this text